sdk/appconfiguration/Azure.Data.AppConfiguration/src/Generated/ConfigurationClient.cs

```diff
@@ -0,0 +1,2437 @@
+            request.Headers.Add("Accept", "application/vnd.microsoft.appconfig.kvset+json, application/json, application/problem+json");
+            return message;
+        }
+
+        internal HttpMessage CreateCheckKeyValuesRequest(string key, string label, string after, string acceptDatetime, IEnumerable<string> select, RequestContext context)
+        {
+            var message = _pipeline.CreateMessage(context, ResponseClassifier200);
+            var request = message.Request;
+            request.Method = RequestMethod.Head;
+            var uri = new RawRequestUriBuilder();
+            uri.Reset(_endpoint);
+            uri.AppendPath("/kv", false);
+            if (key != null)
+            {
+                uri.AppendQuery("key", key, true);
+            }
+            if (label != null)
+            {
+                uri.AppendQuery("label", label, true);
+            }
+            uri.AppendQuery("api-version", _apiVersion, true);
+            if (after != null)
+            {
+                uri.AppendQuery("After", after, true);
+            }
+            if (select != null)
+            {
+                uri.AppendQueryDelimited("$Select", select, ",", true);
+            }
+            request.Uri = uri;
+            if (_syncToken != null)
+            {
+                request.Headers.Add("Sync-Token", _syncToken);
+            }
+            if (acceptDatetime != null)
+            {
+                request.Headers.Add("Accept-Datetime", acceptDatetime);
+            }
+            return message;
+        }
+
+        internal HttpMessage CreateGetKeyValueRequest(string key, string label, string acceptDatetime, IEnumerable<string> select, MatchConditions matchConditions, RequestContext context)
+        {
+            var message = _pipeline.CreateMessage(context, ResponseClassifier200);
+            var request = message.Request;
+            request.Method = RequestMethod.Get;
+            var uri = new RawRequestUriBuilder();
+            uri.Reset(_endpoint);
+            uri.AppendPath("/kv/", false);
+            uri.AppendPath(key, true);
+            if (label != null)
+            {
+                uri.AppendQuery("label", label, true);
+            }
+            uri.AppendQuery("api-version", _apiVersion, true);
+            if (select != null)
+            {
+                uri.AppendQueryDelimited("$Select", select, ",", true);
+            }
+            request.Uri = uri;
+            if (_syncToken != null)
+            {
+                request.Headers.Add("Sync-Token", _syncToken);
+            }
+            if (acceptDatetime != null)
+            {
+                request.Headers.Add("Accept-Datetime", acceptDatetime);
+            }
+            request.Headers.Add("Accept", "application/vnd.microsoft.appconfig.kv+json, application/json, application/problem+json");
+            if (matchConditions != null)
+            {
+                request.Headers.Add(matchConditions);
```

Comment:
You need to regenerate the code again.

Based on https://github.com/Azure/autorest.csharp/pull/2561/files changes, it should generate something like [this](https://github.com/archerzz/autorest.csharp/blob/2f80142d8e85f8d7c89ca803e92ec858e4b5f6a5/src/assets/Generator.Shared/RequestHeaderExtensions.cs#L69).